_episodes/05-cond.md

```diff
@@ -155,8 +155,8 @@ if maximum inflammation in the beginning (day 0) and in the middle (day 20) of
 the study are equal to the corresponding day numbers.
 
 ~~~
-max_inflammation_0 = numpy.max(data, axis=0)[0]
-max_inflammation_20 = numpy.max(data, axis=0)[20]
+max_inflammation_0 = numpy.max(inflamation, axis=0)[0]
```

Comment:
Please undo this (and all other `data` to `inflamation` conversion) change: it has a typo and has not been discussed. We can discuss this change in a separate PR.